roles/cifmw_cephadm/tasks/configure_object.yml

```diff
@@ -38,21 +38,97 @@
     - cifmw_ceph_daemons_layout.rgw_enabled | default(true) | bool
     - swift_in_ctlplane.stdout | bool
 
-- name: Configure object store to use rgw
+- name: Get uuid for project 'service'
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+  ansible.builtin.shell: "set -o pipefail && oc -n {{ cifmw_cephadm_ns }} rsh openstackclient openstack project show service -c id -f value"
```

Comment:
why shell? why pipefail? the `oc` command doesn't pipe anything, does it?

Reply:
++

Reply:
Thanks for your review.
yeah, as it is a simple` oc ` command,  pipefail may not be required here and i agree that `command` is preferred over `shell`.
It is fixed in the latest code.